Dockerfile

```diff
@@ -281,6 +288,7 @@ RUN export COMPOSER_HOME=/composer && \
     composer --no-ansi clearcache && \
     composer --no-ansi require --working-dir=/opt/kimai laminas/laminas-ldap && \
     cp /usr/local/etc/php/php.ini-production /usr/local/etc/php/php.ini && \
+    sed "s/128M/512M/g" /usr/local/etc/php/php.ini-production > /usr/local/etc/php/php.ini && \
```

Comment:
is there only one config for cli and web @tobybatch ?

Reply:
Then we can skip my idea about adding `-d memory_limit=512M bin` to the bin/console calls. 
That is even better 👍